R/table-subscript.R

```diff
@@ -791,9 +791,11 @@ updateSpanIfNecessary <- function(y, x.attributes, evaluated.args) {
         span.df <- mapply(subscriptSpanDF, span.attribute, evaluated.args, SIMPLIFY = FALSE)
         span.df <- Filter(ncol, span.df)
         if (is.null(ncol(y)) && !is.null(span.df$columns)) {
+            nc <- ncol(span.df$columns)
```

Comment:
This is the index of the last column in the column spans data frame, which contains the column names. Perhaps change the variable name to suggest this and add a comment to help readability.